fancyshowcaseview/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.kt

```diff
@@ -92,13 +92,16 @@ class FancyShowCaseView @JvmOverloads constructor(context: Context, attrs: Attri
 
     var queueListener: OnQueueListener? = null
 
+    private inner class QueueListenerProxy : OnQueueListener {
```

Comment:
this is not neccessary.  You can use `queueListener` instance in `FancyShowCaseView`.